library/webpack-plugins/chunk-manifest-plugin.js

```diff
@@ -32,7 +32,7 @@ module.exports = function chunkManifestPlugin() {
         const chunkAssets = {}
         compilation.hooks.chunkAsset.tap(p, (chunk, filename, _) => {
 
-          /* remove if https://github.com/webpack/webpack/issues/7828 is resolved */ if (chunk === "HotModuleReplacementPlugin") return
+          /* remove if https://github.com/webpack/webpack/issues/7828 is resolved */ if (chunk === 'HotModuleReplacementPlugin') return
```

Comment:
De status is nu closed van deze issue.

https://github.com/webpack/webpack/issues/7828

Reply:
Moeten we even een apart ticket voor aanmaken. Dit is zo'n wijziging die we goed moeten testen.